src/Discovery/DiscoveryListView.tsx

```diff
@@ -24,14 +25,23 @@ interface Props {
 }
 
 const DiscoveryListView: React.FunctionComponent<Props> = (props: Props) => {
-  const { searchTerm, config } = props;
+  const { searchTerm, config, selectedSearchableTextFields } = props;
   const [onHoverRowIndex, setOnHoverRowIndex] = useState(null);
   const [onHeightChange, setOnHeightChange] = useState(true);
 
   const scroll = (
     props.config.tableScrollHeight
       ? { scroll: { y: props.config.tableScrollHeight } } : {}
   );
+
+  const isHighlightingEnabled = () => {
+    if (selectedSearchableTextFields.length > 0
+       && !selectedSearchableTextFields.includes('studyPreviewField.field')) {
```

Comment:
it should check if `selectedSearchableTextFields` has the value as defined in `studyPreviewField.field` in portal config, for example https://github.com/uc-cdis/cdis-manifest/blob/master/healdata.org/portal/gitops.json#L284
Not the raw string of `'studyPreviewField.field'`